src/SFA.DAS.Admin.Aan.Web/Models/NetworkEvent/EventOrganiserNameViewModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using SFA.DAS.Admin.Aan.Web.Infrastructure;
+
+namespace SFA.DAS.Admin.Aan.Web.Models.NetworkEvent;
+
+public class EventOrganiserNameViewModel : IEventPageEditFields
```

Comment:
What is the purpose of IEventPageEditFields?

Reply:
|t wraps the common fields PageTitle,PostLink, CancelLink, I use this in all the viewmodels for the controllers about creating events

Reply:
I'll change this to a base class instead in the next story, as discussed